stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/processor/GovActionProposalProcessor.java

```diff
@@ -0,0 +1,71 @@
+package com.bloxbean.cardano.yaci.store.governance.processor;
+
+import com.bloxbean.cardano.client.address.Address;
+import com.bloxbean.cardano.client.util.HexUtil;
+import com.bloxbean.cardano.yaci.core.model.governance.ProposalProcedure;
+import com.bloxbean.cardano.yaci.store.events.EventMetadata;
+import com.bloxbean.cardano.yaci.store.events.GovernanceEvent;
+import com.bloxbean.cardano.yaci.store.events.domain.TxGovernance;
+import com.bloxbean.cardano.yaci.store.governance.domain.GovActionProposal;
+import com.bloxbean.cardano.yaci.store.governance.storage.GovActionProposalStorage;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class GovActionProposalProcessor {
+
+    private final GovActionProposalStorage govActionProposalStorage;
+
+    @EventListener
+    @Transactional
+    public void handleGovernanceAction(GovernanceEvent governanceEvent) {
+        EventMetadata eventMetadata = governanceEvent.getMetadata();
+
+        List<GovActionProposal> govActionProposals = new ArrayList<>();
+        for (TxGovernance txGovernance : governanceEvent.getTxGovernanceList()) {
+            String txHash = txGovernance.getTxHash();
+
+            List<ProposalProcedure> proposalProcedures = txGovernance.getProposalProcedures();
+
+            if (proposalProcedures == null || proposalProcedures.isEmpty()) {
+                continue;
+            }
+
+            int index = 0;
+            for (ProposalProcedure proposalProcedure : proposalProcedures) {
+                GovActionProposal govActionProposal = new GovActionProposal();
+
+                govActionProposal.setTxHash(txHash);
+                govActionProposal.setIndex(index++);
+                govActionProposal.setType(proposalProcedure.getGovAction().getType());
+                govActionProposal.setDescription(proposalProcedure.getGovAction().toString());
```

Comment:
I think if we add description field, we could more clearly describe the content of governance action. For example, if gov_type = PARAMETER_CHANGE_ACTION, description field will contain ProtocolParam update info. I am not sure we should add this field or not but when I refer cardano-db-sync, I think this field may be useful. https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/schema.md#gov_action_proposal

Reply:
Yes, JSON format may be useful and application can parse it if required. Let's try with JSON type field for description. Should we call it description or body/details? Because this is more than simple description :)

Reply:
@Sotatek-HuyLe3a

Reply:
Yes, i will try it and change field name